setup.py

```diff
@@ -5,26 +5,21 @@
 import re
 
 
-v = open(os.path.join(os.path.dirname(__file__), 'ibm_db_sa', '__init__.py'))
+v = open(os.path.join(os.path.dirname(__file__), 'pyodbc_sa', '__init__.py'))
 VERSION = re.compile(r".*__version__ = '(.*?)'", re.S).match(v.read()).group(1)
 v.close()
 
-readme = os.path.join(os.path.dirname(__file__), 'README.rst')
-if 'USE_PYODBC' in os.environ and os.environ['USE_PYODBC'] == '1':
-    require = ['sqlalchemy>=0.7.3']
-else:
-    require = ['sqlalchemy>=0.7.3','ibm_db>=2.0.0']
+readme = os.path.join(os.path.dirname(__file__), 'README.md')
+require = ['sqlalchemy>=0.7.3', 'pyodbc']
     
 
 setup(
-         name='ibm_db_sa',
+         name='pyodbc_sa',
          version=VERSION,
          license='Apache License 2.0',
          description='SQLAlchemy support for IBM Data Servers',
          author='IBM Application Development Team',
          author_email='opendev@us.ibm.com',
-         url='http://pypi.python.org/pypi/ibm_db_sa/',
-         download_url='http://code.google.com/p/ibm-db/downloads/list',
          keywords='sqlalchemy database interface IBM Data Servers DB2 Informix IDS',
```

Comment:
```suggestion
         keywords='sqlalchemy database ibm ibmi db2',
```
I'm not sure keywords really gets used anymore, but we should look at other adapters and see what they use